jerry-debugger/jerry-client.py

```diff
@@ -0,0 +1,22 @@
+import socket
+import sys
+
+port = 5001 
+
+def main():
+	try:
```

Comment:
Do not indent with tab. Use the same indentation as used in other python scripts (eg. tools/build.py uses 4 spaces)